internal/perms/acl.go

```diff
@@ -123,6 +123,21 @@ func (a ACL) Allowed(r Resource, aType action.Type) (results ACLResults) {
 	return
 }
 
+// AllowedActions returns the set of actions out of a given possible set that
+// are allowed given the scope/id/type/pin. Returns nil if no match.
```

Comment:
Mind adding a unit test for this new func?

Reply:
Moved this logic into FetchActionsForId which has tests.